<launch>
	<!-- ========================================================== -->
	<!-- Configuration of the RECORD10SECTRIALSWITHMOVE experiment. -->
    <!--                                                            -->
    <!-- Edit this file to contain the elements of the particular   -->
    <!-- experiment.                                                -->
    <!--                                                            -->
    <!-- Launch this file to run the experiment, as in:             -->
    <!-- roslaunch experiments chase.launch                         -->
    <!--                                                            -->
	<!-- ========================================================== -->

	<!-- ***  Core stuff  *** -->
	<include file="$(find flycore)/launch/flycore.launch" />
	
	<!-- ***  Hardware, Flies, and Robot  *** -->
	<include file="$(find config)/launch/hardware.launch" />
	<include file="$(find config)/launch/robot_0625h_0625w.launch" />
	<include file="$(find config)/launch/flies.launch" />

	<!-- ***  The Experiment  *** -->
	<node pkg="experiments" type="Record10SecTrialsWithMove.py" name="ExperimentRecord10SecTrialsWithMove" />
  
</launch>
